communication/donut.md

```diff
@@ -0,0 +1,5 @@
+<h1>Donut</h1>
+
+One potential downside of being a remote company is that sourceres don't have as many opportunities to have sporadic, informal conversations to get to know each other better.<br>
```

Comment:
sourceres?

Reply:
Should be sourcerers.
sourcerers = source{d} team members